mapit.py

```diff
@@ -0,0 +1,39 @@
+from fabric.api import task, settings, sudo, env, run
+from fabric.tasks import execute
+
+import app
+import nginx
+import puppet
+
+@task
+def update_database():
+    """Update a Mapit database using a new database dump"""
+
+    if len(env.hosts) > 1:
+      exit('This command should only be run on one Mapit machine at a time')
+
+    execute(nginx.gracefulstop)
+    execute(app.stop, 'mapit')
+    sudo('service collectd stop')
```

Comment:
It connects to the database and Postgres won't let you drop a database while connections are still open to it.